docs(link): add good to know comment about scroll behavior (#63152)
vercel/next.js: docs/02-app/02-api-reference/01-components/link.mdx

}
```

> **Good to know**:
>

> > - If `{children}` from a Layout is not present in the current viewport, Next.js will scroll to the top of the `{children}` instead.

Reviewer: ```suggestion
> - If `{children}` from the nested-most Layout is not visible in the viewport, Next.js will scroll to it.
```

> > - If `{children}` from a Layout is not present in the current viewport, Next.js will instead scroll to the top of the `{children}`.

Reviewer: ```suggestion
> - Next.js will scroll to the part of the page that changes as a result of the navigation, if it's not visible in the viewport.
```

Author: I feel like not specifying `{children}` is too vague <picture data-single-emoji=":thinkies:" title=":thinkies:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/thinkies/d164ba479fed9c0d.png" alt=":thinkies:" align="absmiddle"></picture> 

What about `Page` with a link to the [Page](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#pages)?

Author: `Next.js will scroll to the [Page](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#pages) if it's not visible in the viewport upon navigation.`
